parsl/dataflow/dflow.py

```diff
@@ -572,12 +578,14 @@ def cleanup(self):
 
         logger.info("DFK cleanup complete")
 
-    def checkpoint(self):
+    def checkpoint(self, tasks=None):
         """Checkpoint the dfk incrementally to a checkpoint file.
 
         When called, every task that has been completed yet not
         checkpointed is checkpointed to a file.
 
+        Kwargs:
+            - tasks (List of task ids) : List of task ids to checkpoint. Default=None
```

Comment:
clarify what happens in None case (assuming that the intention isn't to checkpoint nothing...)